EosPayload/drivers/science_driver.py

```diff
@@ -0,0 +1,103 @@
+import logging
+import time
+import board
+import busio
+from EosPayload.lib.driver_base import DriverBase
+from EosLib.device import Device
+
+from digitalio import DigitalInOut, Direction, Pull
+from adafruit_pm25.i2c import PM25_I2C
+import Adafruit_BBIO.ADC as ADC
+from adafruit_blinka.microcontroller.am335x import pin
+
+import adafruit_tsl2591
+import adafruit_ltr390
+import adafruit_shtc3
+import adafruit_bmp3xx
+
+class ScienceDriver(DriverBase):
+
+    @staticmethod
+    def enabled() -> bool:
+        return True
+
+    @staticmethod
+    def get_device_id() -> Device:
+        return Device.MISC_3
+
+    @staticmethod
+    def get_device_name() -> str:
+        return "science-driver"
+
+    @staticmethod
+    def read_thread_enabled() -> bool:
+        return True
+
+    def device_read(self, logger: logging.Logger) -> None:
+        i2c = busio.I2C(pin.I2C1_SCL, pin.I2C1_SDA)
+
+        # base sensors
+        sht = adafruit_shtc3.SHTC3(i2c)
+        bmp = adafruit_bmp3xx.BMP3XX_I2C(i2c)
+        ltr = adafruit_ltr390.LTR390(i2c)
+        tsl = adafruit_tsl2591.TSL2591(i2c)
+        
+        # research sensors
+        pm25 = PM25_I2C(i2c)
+        # ADC.setup()
+        logger.info("Starting to poll for science data!")
+
+        count = 0
```

Comment:
^^^ make an __init__ method
define self.count as 0; self.i2c, self.sht, self.bmp, self.ltr, self.tsl, self.pm25 all to None

Then make a setup method and set them to the real values

change all refs to use self.

See:
https://github.com/VIP-LES/EosPayload/blob/main/EosPayload/lib/driver_base.py#L85
https://github.com/VIP-LES/EosPayload/blob/main/EosPayload/lib/driver_base.py#L132